examples/images_contours_and_fields/specgram_demo.py

```diff
@@ -18,7 +17,7 @@
 s2 = 2 * np.sin(2 * np.pi * 400 * t)
 
 # create a transient "chirp"
-mask = np.where(np.logical_and(t > 10, t < 12), 1.0, 0.0)
+mask = (10 < t) & (t < 12)
 s2 = s2 * mask
```

Comment:
I think I'd rather write this as:
```
s2[t <= 10] = 0
s2[12 <= t] = 0
```
(or using an or)

Reply:
good catch, fixed (and rebased)